src/Apps/Feature/Components/__stories__/FeatureHeader.story.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import { omit } from "lodash"
+import { storiesOf } from "storybook/storiesOf"
+import { FeatureHeader } from "../FeatureHeader"
+import { FEATURE_HEADER } from "../__tests__/fixtures"
+
+storiesOf("Apps/Feature", module)
+  .add("FeatureHeader", () => (
+    <div style={{ height: "100vh" }}>
```

Comment:
We haven't used divs in our codebase in a long while, can we stick to box to keep things consistent?